Source/SwiftLintFramework/Rules/RedundantObjcAttributeRule.swift

```diff
@@ -0,0 +1,198 @@
+import Foundation
+import SourceKittenFramework
+
+public struct RedundantObjcAttributeRule: ASTRule, ConfigurationProviderRule {
+
+    public var configuration = SeverityConfiguration(.warning)
+
+    public init() {}
+
+    public static let description = RuleDescription(
+        identifier: "redundant_objc_attribute",
+        name: "Redundant @objc Attribute",
+        description: "Objective C attribute is redundant in declaration.",
```

Comment:
what do you think about: `Objective-C  attribute (@objc) is redundant in declaration.`?